src/java/org/apache/cassandra/metrics/BufferPoolMetrics.java

```diff
@@ -25,24 +25,47 @@
 
 public class BufferPoolMetrics
 {
-    private static final MetricNameFactory factory = new DefaultNameFactory("BufferPool");
+    /** Total number of hits */
+    public final Meter hits;
 
     /** Total number of misses */
     public final Meter misses;
 
-    /** Total size of buffer pools, in bytes */
+    /** Total size of buffer pools, in bytes, including overflow allocation */
     public final Gauge<Long> size;
 
-    public BufferPoolMetrics()
+    /** Total size, in bytes, of active buffered being used from the pool currently + overflow */
+    public final Gauge<Long> usedSize;
```

Comment:
@jasonstack So if there were to be a hole in the bookkeeping logic, one way we might know is that `usedSize` > `size`?

Reply:
if it's over-allocated, then `useSize` > `size`.